src/applications/disability-benefits/view-payments/components/IdentityNotVerified.jsx

```diff
@@ -27,10 +27,8 @@ const IdentityNotVerified = () => {
   );
 
   return (
-    <va-alert status="warning" uswds>
-      <h2 slot="headline" className="vads-u-fontsize--h3">
```

Comment:
`vads-u-fontsize--h3` is not a valid CSS class, I'm guessing they meant `vads-u-font-size--h3`. Since this class hasn't had any effect, I'm just removing it